src/main/java/com/example/bookstoreapp/utils/IdGenerator2.java

```diff
@@ -0,0 +1,14 @@
+package com.example.bookstoreapp.utils;
+
+import java.time.Instant;
+
+public class IdGenerator2 {
+    public static String hostId = "15"; // This is hardcoded here for checking we can define it in properties.
```

Comment:
i just shared the approach, implementation is partially correct. remove this file, we wont use it now.